packages/buttons/src/elements/Anchor.tsx

```diff
@@ -10,12 +10,12 @@ import PropTypes from 'prop-types';
 import { StyledAnchor, StyledExternalIcon } from '../styled';
 
 export interface IAnchorProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
-  /** Apply danger styling */
+  /** Applies danger styling */
   isDanger?: boolean;
   /**
-   * Used when the anchor navigates to an external resource. Applies `target="_blank"`
-   * along with `rel="noopener noreferrer"` to ensure safe
-   * [cross-origin destination links](https://web.dev/external-anchors-use-rel-noopener/).
+   * Attaches `target="_blank"` and `rel="noopener noreferrer"` to ensure
+   * that an anchor that navigates to an external resource is a
+   * safe [cross-origin destination link](https://web.dev/external-anchors-use-rel-noopener/)
```

Comment:
This is moving in the right direction, but it still feels a bit clunky. Have you tried http://www.hemingwayapp.com/ for improving readability? It might be that this just needs to be broken up into two sentences.

Reply:
I did try it with this one, but never broken into two sentences. 

How about: 
Attaches `target="_blank"` and `rel="noopener noreferrer"` to an anchor that navigates to an external resource. This ensures that the anchor is a safe cross-origin destination link.

Reply:
Yeah, much better. Thanks.